student_survey.php

```diff
@@ -0,0 +1,255 @@
+<?php
+require 'survey_query_generators.php';
+header("Content-type: application/json");
+try {
+    $method = "";
+    if (isset($_SERVER['REQUEST_METHOD'])) {
+        $method = $_SERVER['REQUEST_METHOD'];
+        switch ($method) {
+            case "GET":
+                print json_encode(handle_get($_GET));
+                exit();
+            case "POST":
+                print json_encode(handle_post($_POST));
+                exit();
+            default:
+                throw new Exception("Invalid Request");
+        }
+    } else {
+        $error = 'No Request Method Found';
+        throw new Exception($error);
```

Comment:
Why is a method needed?

Reply:
It's used when `$_SERVER['REQUEST_METHOD']` is not found. That means that a GET or POST request is not specified. It's the boilerplate code used from get_info.php.